src/components/admin/hooks/useNewEmployeeFormInfo.js

```diff
@@ -50,6 +50,15 @@ const useNewEmployeeFormInfo = () => {
       gyms: employeeGymList,
     };
 
+    if (!firstName.length || !lastName.length || !email.length || !password.length) {
```

Comment:
Consider


```
const validateFormFields = () => {
  const missingFields = [];
  if (!firstName.length) missingFields.push('First Name');
  if (!lastName.length) missingFields.push('Last Name');
  if (!email.length) missingFields.push('Email');
  if (!password.length) missingFields.push('Password');
  
  return missingFields;
};

const handleSubmit = () => {
  const missingFields = validateFormFields();
  
  if (missingFields.length > 0) {
    dispatch(setNotificationAlert({
      alertType: 'error',
      messageBody: `Missing information: ${missingFields.join(', ')}. Please enter all of the required fields.`,
    }));

    return;
  }

  // etc
};

```

Reply:
Neither this or the other validation restructuring even came to mind, thank you for pointing them out!